web/packages/teleport/src/services/notifications/notifications.ts

```diff
@@ -35,20 +35,21 @@ export class NotificationService {
 
     return api.get(cfg.getNotificationsUrl(params)).then(json => {
       return {
-        notifications:
-          json.notifications.map(notificationJson => {
-            const { id, title, subKind, created, clicked } = notificationJson;
-            const labels = notificationJson.labels || [];
+        notifications: json.notifications
```

Comment:
Couldn't the handler always return an array?

Reply:
we've run into issues before where if the array is empty, it is returned as undefined.